contracts/src/tests_utils/constants.cairo

```diff
@@ -0,0 +1,24 @@
+use starknet::{ContractAddress, contract_address_try_from_felt252};
+
+const TOKEN_MULTIPLIER: u256 = 1000000000000000000;
+const TOKEN0_NAME: felt252 = 'TOKEN0';
+const TOKEN1_NAME: felt252 = 'TOKEN1';
+const TOKEN2_NAME: felt252 = 'TOKEN2';
+const SYMBOL: felt252 = 'SYMBOL';
+const MINIMUM_LIQUIDITY: u256 = 1000;
+
+fn DEPLOYER() -> ContractAddress {
+    contract_address_try_from_felt252('DEPLOYER').unwrap()
```

Comment:
just fyi - no need for change because it's a small nit, but prefer doing

```
'DEPLOYER'.try_into().unwrap()
```

as it saves you the explicit import of the function. 
If you're writing this in source code, you can use `contract_address_const` which is more optimized

Reply:
Interesting, thanks for the information. So, instead of declaring these functions that simulate being constants, it is more efficient to use it directly in the test like `contract_address_const<'DEPLOYER'>()`?

Reply:
> more efficient to use it directly in the test like contract_address_const<'DEPLOYER'>()?
in the test we don't care and I personnaly prefer 'deployer'.try_into().unwrap() as I don't need to take care of importing anything. In source, it's more optimized yes.